.github/workflows/update_org_members.yml

```
@@ -0,0 +1,28 @@
+# GitHub Actions workflow for auto-inviting org members to human intervention.
+#
+# The file coordinate_package_policies.json contains the the thresholds for
+# triggering addition for each of the coordinate packages.
+
+name: Invite organzation members based on merged PRs
+
+on:
+  schedule:
+    # Run once a day at 22:14 UTC
```

That's a great time, but I'm assuming it's arbitrary? If so, maybe just add a note to remind future us or other who see this that this isn't a special time 😅 
```suggestion
    # Run once a day at 22:14 UTC (arbitrary time)
```